contract/models/res_partner.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    contract_count = fields.Integer(
+        compute='_compute_contract_count',
+        string='Contracts')
+
+    def _compute_contract_count(self):
+        Contract = self.env['account.analytic.account']
+        today = fields.Date.today()
+        for partner in self:
+            partner.contract_count = Contract.search_count([
+                ('recurring_invoices', '=', True),
```

Comment:
Is it needed to filter by this criteria?

Reply:
It seems that you still do not know
that users are capable :wink: 

What is the problem?

Reply:
No, it's just my ignorance on the deeper concepts of the module :smile: Can't a contract with no invoices be considered valid?

Reply:
https://github.com/OCA/contract/pull/107/files#diff-47b79491db78ebebcb372014ce9c7bf3R39

Reply:
It's just ambiguous naming

Reply:
I agree.

https://github.com/odoo/odoo/blob/6140300da011d9eb4196b780b011bbc87fe8cae0/addons/account_analytic_analysis/account_analytic_analysis.py#L528
https://github.com/OCA/contract/blob/35d6358471c071ecdb17d936c0b5e11ee28a86ee/contract/models/contract.py#L114